### PR TITLE
FHOPS-4424 update pr templating for repository rhmap-backups.

### PR DESCRIPTION
**Pull request by Nanyte25**
https://issues.jboss.org/browse/FHOPS-4424

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Jira link(s)
```
Comment (pmccarthy) [inline .github/pull_request_template.md:1]: Bearing in mind that this is a public repository, we shouldn't be adding links to Jiras in PRs as only Red Hat associates would have access to them

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# What
+
+
+
+# Why
+
+
+
+
+# How
```
Comment (pmccarthy) [inline .github/pull_request_template.md:14]: Would "How" not be covered in the Verification steps?

**Comment by pmccarthy** [comment]
As per my previous comment, this repository is public so referencing Jira tickets in the subject header and description won't mean much to anyone outside of Red Hat. Perhaps you could use the template in this PR to update the description here :)

**Comment by Nanyte25** [comment]
In that case closing pr. Thanks @pmccarthy

**Pull request closed without merging**